app/RssSync.php

```diff
@@ -40,6 +40,6 @@ private function transformItem($item)
 
     private function contentToAbstract($content)
     {
-        return mb_substr($content, 0, 225) . '...';
+        return mb_substr($content, 0, 225).'...';
```

Comment:
Can you revert this? we prefer spaces around our concatenators :)
```suggestion
        return mb_substr($content, 0, 225) . '...';
```